### ISSUE TITLE
can i use yolo_v2 (training, detecting) without batch-normalization???

### ISSUE THREAD

**Issue by donggru11** [pjreddie/darknet#976]
im studying deep learning with yolo(version 2).
my own database is an almost same view, so batch-normalization is un-needed.
On the contrary, it increases the computation time.
So i was trying to train my own model without batch-normalization.
[ batch_normalize=0 ] 
But it cannot convergence. training doesn't work at all if i reduce learning-rate very low.

can't we train yolo-v2 without batch-normalization?
or any special methods to omit batch-normalization????
